ulid/__init__.py

```diff
@@ -70,7 +70,9 @@ class ULID:
     def __init__(self, value: bytes | None = None) -> None:
         if value is not None and len(value) != constants.BYTES_LEN:
             raise ValueError("ULID has to be exactly 16 bytes long.")
-        self.bytes: bytes = value or ULID.from_timestamp(time.time()).bytes
+        self.bytes: bytes = (
```

Comment:
BUG.

Previous function took floating point of in seconds
now
you give integer milliseconds


PLEASE add tests.

Reply:
Can you quickly describe what the BUG is? The classmethod `ULID.from_timestamp` did already before allow an `int` or a `float` to be passed as value. The only difference is IMO that the `ULID` constructor now uses an `int` by default (from `time.time_ns()`) which used to be `float` (from `time.time()`). We could add an additional test though.

Reply:
Earlier in interpreted passed number as seconds. (i.e. to extract milliseconds it multiplied it by 1000)

Now, you pass milliseconds directly, but as integer. If `ULID.from_timestamp` still multiplies by 1000 (I guess yes) - there is a bug, since extraxting milliseconds inside it would give exactly 0 every time. Does not it ?

Reply:
Sorry, I did not see code. `if isinstance(value, float):` - is a very bad thing. For example after reading JSON sometimes a number can be float or int, depending on moon phase. I wouldn't recommend do such things in python code (i.e. make assumptions on units depending on type of value)

Reply:
Sorry, I don't see how your wisdom around how JSON values can be parsed, is in any way related to actual "issue". If you have an actual improvement proposal feel free to create a pull-request. Otherwise, I find it quite disturbing to jump out of nowhere and scream "BUG." (in all caps), without even looking at the code before.

Reply:
Sorry, you are right, my bad. The caps-locked term is from Linux kernel https://lkw.readthedocs.io/en/latest/doc/06_kernel_bug_reporting.html#bug-bug-on-dump-stack-panic-example

I can not propose a change, because it's part of your public API. So, changing it would affect compatibility across versions.

So, nothing to change. No bugs, actually. I just wanted to help.